CMPUT_301_Project/app/src/main/java/com/example/cmput_301_project/HabitFragments.java

```diff
@@ -63,6 +70,31 @@ public void onAttach(Context context){
         }
     }
 
+    public boolean checkFields(){
+        String name = habitTitle.getText().toString();
+        String reason = startDate.getText().toString();
+        int weekdays = 0;
+        if (monday.isChecked() == true)
+            weekdays = weekdays + 1;
+        if (tuesday.isChecked() == true)
+            weekdays = weekdays + 2;
+        if (wednesday.isChecked() == true)
+            weekdays = weekdays + 4;
+        if (thursday.isChecked() == true)
+            weekdays = weekdays + 8;
+        if (friday.isChecked() == true)
+            weekdays = weekdays + 16;
+        if (saturday.isChecked() == true)
+            weekdays = weekdays + 32;
+        if (sunday.isChecked() == true)
+            weekdays = weekdays + 64;
```

Comment:
why are we adding up weekdays here? Wouldn't it be simpler to just have a 'hasWeekdaySelected' bool instead since we only care about if a weekday is checked, not which one(s)?